src/main/paradox/docs/api/assets/acls-add.json

```diff
@@ -0,0 +1,35 @@
+{
+  "acl": [
+    {
+      "permissions": [
+        "read"
+      ],
+      "identity": {
+        "@type": "Group",
+        "realm": "realm"
```

Comment:
missing group; I'd recommend dropping the `@type: Group` and just put the group value.